buildbot/tmp_plot_patches.py

```
@@ -89,15 +89,15 @@ def draw_cube(xm, ym, zm, xp, yp, zp, color, alpha):
 
 interfaces_dic = {}
 
-for k in patch_dic.keys():
+for k in patch_dic:
     interfaces_dic[k] = []
 
 interf_filelist = ["interfaces_" + str(iteration) + "_node" + str(n) for n in range(4)]
 it = 0
 for a in interf_filelist:
     fil = open(a, "r")
```

Ruff (0.16.0)

[warning] Use a context manager for opening files

(SIM115)